aerleon/lib/pcap.py

```diff
@@ -228,7 +234,9 @@ def JoinConditionals(condition_list, operator):
         res = '(%s)' % (op.join(condition_list))
         return res
 
-    def _GenerateAddrStatement(self, addrs, exclude_addrs):
+    def _GenerateAddrStatement(
+        self, addrs: List[Union[str, IPv6, IPv4]], exclude_addrs: List[Any]
```

Comment:
`exclude_addrs` should have the same type as `addrs`, right?

Reply:
Yup, think these were missed during the late night coding.